zebra-network/src/peer/handshake.rs

```diff
@@ -510,17 +572,46 @@ where
                                 // Heartbeats are checked internally to the
                                 // connection logic, but we need to wait on the
                                 // response to avoid canceling the request.
-                                match rx.await {
-                                    Ok(_) => tracing::trace!("got heartbeat response"),
-                                    Err(_) => {
-                                        tracing::trace!(
+                                //
```

Comment:
```suggestion

```

Reply:
I think the extra blank comment line improves readability - check the split diff view.